### ISSUE TITLE
Unexpected result from denseToDecimal

### ISSUE THREAD

**Issue by ulfl** [k0001/safe-money#39]
I get the following results from `denseToDecimal` (using version 0.7 of safe-money):
```haskell
> denseToDecimal Round False (Just ',') '.' 2 (1 % 1) (fromJust $ dense (1%1) :: Dense "USD")
Just "1.00"
> denseToDecimal Round False (Just ',') '.' 2 (1 % 1) (fromJust $ dense (1%10) :: Dense "USD")
Just "0.10"
> denseToDecimal Round False (Just ',') '.' 2 (1 % 1) (fromJust $ dense (1%100) :: Dense "USD")
Just "0.10"
```
Was expecting `0.01` in the last result.

**Comment by k0001** [comment]
Hm, yes, I managed to reproduce this. There seems to be an issue when rendering values less than `0.10` USD. I'll investigate. Thanks.

Note that this is just a text rendering issue, not an issue on `Dense` or `Discrete`.

**Comment by k0001** [comment]
@ulfl this has been fixed in `0.7.1`. The issue was that the necessary padding `0`s where being added after the `1`, rather than before the `1`. I added many tests covering this case as well.

**Comment by ulfl** [comment]
@k0001 Thanks for fixing and for an excellent library.
